### DISCUSSION TITLE
Sticky table headers

### DISCUSSION THREAD

**Discussion by aaronhudon** [tui-cs/Terminal.Gui#2211] [Q&A] [answered]
Has there been any mention/discussion of having the TableView support a "sticky" header, whereby the contents of the table will scroll but the headers remain visible at all times?

**Comment by tznind** [comment, marked as answer]
Hi thanks for your interest!  This is indeed an option.  You can use `tableView.Style.AlwaysShowHeaders = true` to keep the headers always visible.

**Reply by tznind** [reply]
@aaronhudon just incase you weren't aware just thought I'd highlight that this setting is also available in the [TerminalGuiDesigner](https://github.com/gui-cs/TerminalGuiDesigner) if you are using that:

![image](https://user-images.githubusercontent.com/31306100/201061144-4924a399-cd43-4767-bf66-6d0b89da7893.png)
